components/user/UserProfile.js

```diff
@@ -52,7 +52,7 @@ function UserProfile({ BASE_URL, data }) {
         <div className="flex flex-col self-center gap-3">
           <h1 className="text-3xl font-bold">{data.name}</h1>
           <div className="flex md:w-full gap-2 mx-auto text-xl">
-            {data.socials?.map((social) => (
+            {data.links?.map((social) => (
```

Comment:
Not all `links` are pinned as `socials`

Reply:
The socials are not sorted properly in `getUserApi`.